src/test/groovy/com/rabbitmq/http/client/ReactorNettyClientSpec.groovy

```diff
@@ -407,7 +407,7 @@ class ReactorNettyClientSpec extends Specification {
 
         then: "the result is empty and the response handling code has been called"
         vhost.hasElement().block() == false
-        called.get() == true
+        waitAtMostUntilTrue(5, { called.get()})
```

Comment:
@simonbasle @smaldini  The onResponse callback seems to be called after getting the response. This wasn't the case with Reactor Netty 0.8.0.RELEASE. I had to change this test assertion to poll the flag set in the callback after getting the response. 
cc @violetagg